src/server/api/routers/exhibitor.ts

```diff
@@ -347,7 +343,7 @@ export const exhibitorRouter = createTRPCRouter({
               type: input.type,
             },
             data: {
-              value: input.value,
+              value: input.value.length == 0 ? ["None"] : input.value,
```

Comment:
Why special case the empty case? Can't we just leave it as an empty array?

Seems to me like the only change that has to happen is to remove the if-statement above (which you did)

Reply:
True, changed this to be the empty array instead.